R/rdeephaven/R/operation_control.R

```diff
@@ -1,8 +1,7 @@
-#' @description
-#' An OperationControl defines the control parameters of some UpdateByOps used in an update_by table operation.
-#' The UpdateByOps can use OperationControl to handle erroneous data are `ema_tick`, `ema_time`, `emmax_tick`,
-#' `emmax_time`, `emmin_tick`, `emmin_time`, and `ems_time`.
-#' Note that OperationControl should not be instantiated directly by user code, but rather `by op_control()`.
+# An OperationControl defines the control parameters of some UpdateByOps used in an update_by table operation.
```

Comment:
You have quoted with backticks in other parts of the documentation for language-level names (eg, parameter names to functions).  Perhaps you want to backtick-quote `OperationControl` ?